README.md

```diff
@@ -49,90 +37,120 @@ app.use(errorHandler({
+### Customizing log format
 
-When not enabled, it only sends the error back in the response.
+**Express** 
 
-Customization of the log format is intentionally not allowed. If you would like
-to use a different format/logger, disable this option and add your own custom
-error-handling middleware.
+To use a different log format, add your own custom error-handling middleware then disable `errorHandler.log`. 
+For example, in an Express application:
 
 ```js
 app.use(myErrorLogger());
 app.use(errorHandler({ log: false }));
 ```
 
-## Migration to strong-error-handler for existing LoopBack applications
+In general, add `strong-error-handler` as the last middleware function, just before calling `app.listen()`.
 
-1. In package.json dependencies, remove ` "errorhandler": "^x.x.x”,`
-1. Run `npm install --save strong-error-handler`
-1. In `./server/config.json`, remove:
+**LoopBack**
 
-  ```
-  "errorHandler": {
-    "disableStackTrace": false
-  }
-  ```
-  and replace it with `"handleErrors": false`.
-1. In `server/middleware.json`, remove
+For LoopBack applications, put custom error-logging middleware in a separate file; for example, `server/middleware/error-logger.js`:
 
-  ```
-    "final:after": {
-    "loopback#errorHandler": {}
-  }
-  ```
-  and replace it with:
-  ```
+```
+module.exports = function(options) {
+  return function logError(err, req, res, next) {
+    console.log('unhandled error' ,err);
+    next(err);
+  };
+};
+```
+
+Then in `server/middleware.json`, specify your custom error logging function as follows:
+
+```
+{
+  // ...
   "final:after": {
-    "strong-error-handler": {}
-  }
-  ```
-1. In the `./server`, delete `middleware.production.json`.
-1. In the `./server`, create `middleware.development.json` containing:
+    "./middleware/error-logger": {},
+    "strong-error-handler": {
+      "params": {
+        log: false
+      }
+    }
+}
+```
+
+The default `middleware.development.json` file explicitly enables logging in strong-error-handler params, so you will need to change that file too.
 
-  ```
-  {
+## Migration from old LoopBack error handler
+
+NOTE: This is only required for applications scaffolded with old versions of the `slc loopback` tool.
+
+To migrate a LoopBack 2.x application to use `strong-error-handler`:
+
+1. In `package.json` dependencies, remove `"errorhandler": "^x.x.x”,`
+1. Install the new error handler by entering the command:
+    <pre>npm install --save strong-error-handler</pre>
+1. In `server/config.json`, remove:
+    <pre>
+    "errorHandler": {
+      "disableStackTrace": false
+    }</pre>
+  and replace it with `"handleErrors": false`.
+1. In `server/middleware.json`, remove:
+    <pre>
+    "final:after": {
+      "loopback#errorHandler": {}
+    }</pre>
+  and replace it with:
+    <pre>
+    "final:after": {
+      "strong-error-handler": {}
+    }</pre>
+1. Delete `server/middleware.production.json`.
+1. Create `server/middleware.development.json` containing:
+  <pre>
+  ...
```

Comment:
I think the dots are not necessary here, because we are creating a new file, thus the code snippet here should show the full content of this new file.